marklogic-data-hub/src/main/resources/ml-modules/root/com.marklogic.smart-mastering/survivorship/merging/options.xqy

```diff
@@ -632,7 +632,7 @@ declare function merge-impl:compile-merge-options(
         map:map()
       else ()
     (: Gather target Entity Type information :)
-    let $target-entity := $merge-options/(*:target-entity|targetEntity|targetEntityType) ! fn:string(.)
+    let $target-entity := fn:head(($merge-options/targetEntityType,$merge-options/(*:target-entity|targetEntity))) ! fn:string(.)
```

Comment:
Is there a common library module that would make sense for this function to live in so that both options-impl and options.xqy can use it? Like the util.xqy module in com.marklogic.smart-mastering?

Reply:
This has been moved into the util.xqy and a test has been added for that function